coins/ethereum/tests/contract.rs

```diff
@@ -0,0 +1,74 @@
+#[cfg(test)]
+mod tests {
```

Comment:
You don't need the mod tests if this is under tests/. You just have `mod tests;` in `lib.rs`, which is cfg test, and then `tests/mod.rs` with the tests in that file/`mod contract;` to keep tests under tests/contract.rs.

Reply:
updated